### PR TITLE
Feat: Rename Message to Block

### PR DESCRIPTION
**Pull request by brancoder**
# Description of change

Renaming:
- Message to Block
- UnlockBlocks to Unlocks
- FeatureBlocks to Features
- parentMessageIds to parents
- childrenMessageIds to children
- confirmedMerkleRoot to inclusionMerkleRoot

Fixes: #870 

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Change checklist

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/iota/docs/interfaces/IMilestonePayload.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Milestone payload.
 - [index](IMilestonePayload.md#index)
 - [timestamp](IMilestonePayload.md#timestamp)
 - [previousMilestoneId](IMilestonePayload.md#previousmilestoneid)
-- [parentMessageIds](IMilestonePayload.md#parentmessageids)
+- [parentBlockIds](IMilestonePayload.md#parentblockids)
```
Comment (lzpap) [inline packages/iota/docs/interfaces/IMilestonePayload.md]: On API level these will be called simply `parents`.

File: packages/iota/docs/interfaces/IBlock.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
 ## Table of contents
 
 ### Properties
 
-- [protocolVersion](IMessage.md#protocolversion)
-- [parentMessageIds](IMessage.md#parentmessageids)
-- [payload](IMessage.md#payload)
-- [nonce](IMessage.md#nonce)
+- [protocolVersion](IBlock.md#protocolversion)
+- [parentBlockIds](IBlock.md#parentblockids)
```
Comment (lzpap) [inline packages/iota/docs/interfaces/IBlock.md]: `parents`

File: packages/iota/docs/interfaces/IChildrenResponse.md [hunk 3]
```diff
@@ -6,18 +6,18 @@ List of children found.
 
 ### Properties
 
-- [messageId](IChildrenResponse.md#messageid)
+- [blockId](IChildrenResponse.md#blockid)
 - [maxResults](IChildrenResponse.md#maxresults)
 - [count](IChildrenResponse.md#count)
-- [childrenMessageIds](IChildrenResponse.md#childrenmessageids)
+- [childrenBlockIds](IChildrenResponse.md#childrenblockids)
```
Comment (lzpap) [inline packages/iota/docs/interfaces/IChildrenResponse.md]: This field is also just gonna be called `children`.

File: packages/iota/docs/interfaces/IMilestonePayload.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ Milestone payload.
...
 - [confirmedMerkleRoot](IMilestonePayload.md#confirmedmerkleroot)
```
Comment (lzpap) [inline packages/iota/docs/interfaces/IMilestonePayload.md]: `confirmedMerkleRoot` will be `inlcusionMerkleRoot`

File: packages/iota/docs/interfaces/IBlockMetadata.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
+# Interface: IBlockMetadata
+
+Response from the metadata endpoint.
+
+## Table of contents
+
+### Properties
+
+- [blockId](IBlockMetadata.md#blockid)
+- [parentBlockIds](IBlockMetadata.md#parentblockids)
```
Comment (lzpap) [inline packages/iota/docs/interfaces/IBlockMetadata.md]: also `parents`

**Review by lzpap — APPROVED** [review]

**Review by msarcev — APPROVED** [review]

**Pull request merged**